docs/trusted-web-activity-browser-support.md

```diff
@@ -0,0 +1,42 @@
+Trusted Web Activity Browser Support
+====================================
+
+**Important:** This is an unnofficial list of Trusted Web Activity support across
```

Comment:
If we're in a repo owned by "GoogleChrome", can this really be unofficial?

Why are we saying it's unofficial? In case there are mistakes?

Reply:
We can definitely keep the Chrome details updated, but not other vendors. Maybe we can word this differently?

Reply:
How about "We do our best to keep this up to date - if you spot anything missing or incorrect, please file an issue at XX".

Reply:
Updated